_posts/2023-03-08-february-rubygems-updates.md

```diff
@@ -19,28 +19,32 @@ The following improvements and fixes are included in these releases (see the cha
 - restored better error message when locked ref does not exist - [#6356](https://github.com/rubygems/rubygems/pull/6356).
 - avoid gem crashing when installing from a corrupted lockfile - [#6355](https://github.com/rubygems/rubygems/pull/6355).
 - fixed crash in pub grub involving empty ranges - [#6365](https://github.com/rubygems/rubygems/pull/6365).
-- made gemspec file generated by `bundle gem` properly exclude itself from packaged gem - [#6339](https://github.com/rubygems/rubygems/pull/6339).
-- fixed changelog generation - [#6381](https://github.com/rubygems/rubygems/pull/6381).
-- preserved `bundler-setup-relative` paths if the `:path` option is set to relative in standalone setup - [#6327](https://github.com/rubygems/rubygems/pull/6327).
+
+Other improvements we worked on during this month, but that did not make it into a February release are:
+
+- added an experimental feature for `gem exec` command to run executables from gems that may or may not be installed - [#6309](https://github.com/rubygems/rubygems/pull/6309).
 - safe load all marshaled data - [#6384](https://github.com/rubygems/rubygems/pull/6384).
+- Make gemspec file generated by bundle gem properly exclude itself from packaged gem - [#6339](https://github.com/rubygems/rubygems/pull/6339).
+- preserved `bundler-setup-relative` paths if the `:path` option is set to relative in standalone setup - [#6327](https://github.com/rubygems/rubygems/pull/6327).
 
 In February, RubyGems gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-02-01%7D...master@%7B2023-02-28%7D), contributed by 16 authors. There were 1,767 additions and 234 deletions across 106 files.
 
 ## RubyGems.org News
 
-This month, we made a lot of progress working on the backend admin dashboard. We implemented robust auditing of all changes, and added support for resetting users MFA, blocking a user, and deleting webhooks. 
+This month, we made significant progress on the backend admin dashboard. We implemented robust auditing of all changes and added support for resetting user's MFA, blocking a user, and deleting webhooks. 
 
-The dependency API deprecation has been [announced](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html), and we have a plan for brownouts & turning off the endpoint entirely.
+We have [announced](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html) the deprecation of the dependency API, and we plan to implement brownouts and remove the endpoint entirely. We migrated all RDS instances to Terraform and tested the migration of managed node groups.
```

Comment:
We migrated all RDS instances to be managed by Terraform and tested the migration of managed node groups on rubygems.org EKS cluster.